retrieval/scrape.go

```diff
@@ -297,6 +307,7 @@ func (s *targetScraper) scrape(ctx context.Context, ts time.Time) (model.Samples
 	}
 	req.Header.Add("Accept", acceptHeader)
 	req.Header.Set("User-Agent", userAgentHeader)
+	req.Header.Set("Scrape-Timeout-Seconds", fmt.Sprintf("%0.1f", s.timeout.Seconds()))
```

Comment:
There are some using sub-second timeouts, I'd remove the precision and do a simple %f